tests/func/test_data_cloud.py

```diff
@@ -73,29 +73,31 @@ def test(self):
             self._test_cloud(config, cl)
 
 
-class TestDataCloudBase(TestDvc):
+class DataCloudBaseTestCase(TestDvc):
     def _get_cloud_class(self):
         return None
 
-    def _should_test(self):
+    @staticmethod
+    def should_test():
         return False
 
-    def _get_url(self):
-        return ""
+    @staticmethod
+    def get_url():
+        return NotImplementedError
```

Comment:
I still want to keep this, so, if `get_url` is not defined in inherited classes, we will know. :)

Reply:
@skshetry makes sense! 👍